src/react/components/_academics/AcademicSummary/LawSection.js

```diff
@@ -13,6 +13,7 @@ const LawSection = ({
   units,
   canViewGrades
 }) => {
+  let formattedlawUnits = lawUnits ? lawUnits : '0.0'
```

Comment:
@frankiepun - Why not just use OR (`||`)?

```javascript
let formattedLawUnits = lawUnits || '0.0'
```

Reply:
thanks for the suggestion. I learned a new trick.I have updated the code.